powerlevel9k.zsh-theme

```diff
@@ -1006,18 +1015,36 @@ $(print_icon 'MULTILINE_SECOND_PROMPT_PREFIX')"
     RPROMPT_PREFIX=''
     RPROMPT_SUFFIX=''
   fi
-
   if [[ "$POWERLEVEL9K_DISABLE_RPROMPT" != true ]]; then
-    RPROMPT="$RPROMPT_PREFIX%f%b%k$(build_right_prompt)%{$reset_color%}$RPROMPT_SUFFIX"
+    # kill previous async
+    if [[ "${ASYNC_PROC}" != 0 ]]; then
+      kill -TERM $ASYNC_PROC >/dev/null 2>&1
+    fi
+    async &!
+    ASYNC_PROC=$!
   fi
 }
 
+tidy() {
+  rm $socket
```

Comment:
this should be `rm -f`, since some implementations force a confirm otherwise (e.g. os x)